events/2016/boston.html

```diff
@@ -0,0 +1,130 @@
+          </li>
+          <li>
+            <a href="#help">Volunteer</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <section class="jumbo">
+      <div class="container">
+        <h1 class="logo center-text main-logo">NodeTogether</h1>
+        <div class="brought-to-you">
+          <h3>
+            Brought to you by
+          </h3>
+          <img id="npm" src="/images/npm-logo.svg" alt="npm">
+        </div>
+        <h1 class="subtitle center-text">Boston, MA, USA</h1>
+          <h3 class="center-text">Apply Now!</h3>
+          <div class="calls-to-action center-text">
+            <a href="http://goo.gl/forms/bzUiKynzGK" class="btn">Student</a>
+            <a href="http://goo.gl/forms/fFzciI8lv5" class="btn">Mentor</a>
+            <a href="http://goo.gl/forms/wBX2XS0cd9" class="btn">Sponsor</a>
+          </div>
+     </section>
+      </div>
+    </section>
+    <section id="details">
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <h3>Date: August 23, 2016</h3>
+            <h3>Time: 5:00pm - 10:00pm</h3>
+            <h3>Location:</h3>
+            <address>
+              Bocoup,
+              201 South Street, Floor 1,
+              Boston, MA 02111
+              United States of America
+            </address>
+          </div>
+          <div class="col">
+            <a href="https://www.google.com/maps/place/Bocoup/@42.350116,-71.060129,17z/data=!3m1!4b1!4m5!3m4!1s0x89e37a78e33586ab:0xc04fecb4021b6141!8m2!3d42.350116!4d-71.057935">
+              <img src="/images/boston-map.png" alt="boston map" />
+            </a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="apply">
+      <div class="container">
+        <h2 class="center-text">Join us!</h2>
+        <p>The ideal student for a NodeTogether event is a new or junior developer who is interested in getting started with Node.js. We don’t expect you to have any programming experience and will be spending a lot of time getting a development environment set up.</p>
+       <p>Additionally, we are particularly interested in people from underrepresented groups, including but not limited to, women, people of color, people with mental and/or physical disabilities, and LGBTQ and non-binary people.</p>
+        <div class="center-text">
+          <a href="http://goo.gl/forms/bzUiKynzGK" class="btn">Apply as a student</a>
+        </div>
+      </div>
+    </section>
+    <section id="sponsors">
+      <div class="container center-text">
+        <a href="http://npmjs.com">
+          <div class="sponsor">
+            <h2>Primary Sponsor</h2>
+            <img src="/images/npm-logo.svg" alt="npm" style="max-width: 225px"/>
+          </div>
+        </a>
+        <a href="https://bocoup.com/">
+          <div class="sponsor">
+            <h2>Venue Sponsor</h2>
+            <img src="/images/bocoup.png" alt="Bocoup" style="max-width: 300px"/>
```

Comment:
Wrong file name for the Bocoup logo! No worries, though - just check the name of the logo in the `/images` folder. You can change it to the correct file name _here_, then commit and push. The commit will be automatically added here!